storybook/src/docs/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -17,6 +23,29 @@ import * as React from "react";
 import { apolloStoryDecorator } from "../../../apollo-story.decorator";
 import { storyRouterDecorator } from "../../../story-router.decorator";
 
+type Launch = {
+    id: string;
+    mission_name: string;
+    launch_date_local: Date;
```

Comment:
Is there a reason for using `snake_case` here instead of `camelCase`?

Reply:
This definition comes from here: https://github.com/vivid-planet/comet/blob/main/storybook/.storybook/mocks/handlers.ts#L19